core/print.c

```diff
@@ -1333,7 +1354,7 @@ static int isom_print_sample_description_extesion( FILE *fp, lsmash_file_t *file
     {
         lsmash_box_type_t type;
         int (*print_func)( FILE *, lsmash_file_t *, isom_box_t *, int );
-    } print_description_extension_table[33] = { { LSMASH_BOX_TYPE_INITIALIZER, NULL } };
+    } print_description_extension_table[64] = { { LSMASH_BOX_TYPE_INITIALIZER, NULL } };
```

Comment:
I noticed that a few other tables had their size rounded up to the next power of 2, so I just bumped this to 64 so that it doesn't need to be incremented every time, assuming it has a negligible impact on memory usage and performance.